### DISCUSSION TITLE
What is the new feature "."?

### DISCUSSION THREAD

**Discussion by chunliu** [cse-labs/kubernetes-in-codespaces#31] [Q&A] [answered]
In the README, it mentioned a new GitHub feature ".". What is it? Can you please share a link of it?

**Comment by bartr** [comment, marked as answer]
In any GitHub repo, press the "." and you get the new Web Editor view. This is an amazingly simple experience for doc updates and small edits. You can make your changes and do a PR without ever leaving the Web Editor. Super cool!
